app/views/plant_parts/index.html.haml

```diff
@@ -9,7 +9,8 @@
     - if plant_part.crops.empty?
       No crops are harvested for this plant part (yet).
     - else
-      != plant_part.crops.map { |c| link_to(c, c) }.join(", ")
+      - plant_part.crops.limit(100).each do |crop|
```

Comment:
Maybe worth sticking up into the controller the "decide how many things to show" parts? Not that we've done it here, but if we were doing pagination for this list it'd make sense there.

Reply:
yeah, i agree. Will move

Reply:
This page doesn't seem to have any links to it.... Maybe it serves no useful purpose?